src/assets/styles/colours.css

```diff
@@ -242,6 +252,6 @@ html {
     --colour-service-accent-1: var(--colour-send-primary-accent-1);
     --colour-service-accent-2: var(--colour-send-primary-accent-2);
   }
-
+  */
```

Comment:
Do we need those for reference?

Reply:
Once someone confirms the colours they can be removed. 😅